src/commands/generateSdk.ts

```diff
@@ -301,19 +298,20 @@ async function generateRemoteSdkHandler(
         );
         debugLogger.debug("Package json is: " + packageJson);
         debugLogger.debug("Start Compiling sdk");
-        await compileSdk(sdkPath, packageJson, language as Language, false, "");
+        await compileSdk(sdkPath, packageJson, language as Language, false, "", false);
```

Comment:
Usually, when the passed function arguments can't be understood by reading the passed values, a comment is placed before them.


```suggestion
        await compileSdk(
            sdkPath,
            packageJson,
            language as Language,
            /* publish= */ false,
            /* outDir= */ "",
            /* overwriteIfExists= */ false,
        );
```